lib/cli/run.js

```diff
@@ -0,0 +1,403 @@
+        group: GROUPS.RULES,
+        requiresArg: true,
+        type: 'string'
+      },
+      grep: {
+        alias: 'g',
+        coerce: value => (!value ? null : value),
+        description: 'Only run tests matching this string or regexp',
+        group: GROUPS.FILTERS,
+        requiresArg: true,
+        type: 'string'
+      },
+      growl: {
+        alias: 'G',
+        description: 'Enable Growl notifications',
+        group: GROUPS.OUTPUT,
+        type: 'boolean'
+      },
+      interfaces: {
+        description: 'List built-in user interfaces & exit',
+        type: 'boolean'
+      },
+      invert: {
+        alias: 'i',
+        description: 'Inverts --grep and --fgrep matches',
+        group: GROUPS.FILTERS,
+        type: 'boolean'
+      },
+      opts: {
+        default: './test/mocha.opts',
+        description: 'Path to `mocha.opts`',
+        group: GROUPS.CONFIG,
+        normalize: true,
+        requiresArg: true
+      },
+      recursive: {
+        description: 'Look for tests in subdirectories',
+        group: GROUPS.FILES,
+        type: 'boolean'
+      },
+      reporter: {
+        alias: 'R',
+        default: 'spec',
+        description: 'Specify reporter to use',
+        group: GROUPS.OUTPUT,
+        requiresArg: true,
+        type: 'string'
+      },
+      reporters: {
+        description: 'List built-in reporters & exit',
+        type: 'boolean'
+      },
+      require: {
+        alias: 'r',
+        default: [],
+        defaultDescription: '(none)',
+        description: 'Require module',
+        group: GROUPS.FILES,
+        requiresArg: true,
+        type: 'array'
+      },
+      retries: {
+        defaultDescription: '0',
+        description: 'Retry failed tests this many times',
+        group: GROUPS.RULES,
+        type: 'number'
+      },
+      slow: {
+        alias: 's',
+        default: 75,
+        description: 'Specify "slow" test threshold (in milliseconds)',
+        group: GROUPS.RULES,
+        type: 'number'
+      },
+      sort: {
+        alias: 'S',
+        description: 'Sort test files',
+        group: GROUPS.FILES,
+        type: 'boolean'
+      },
+      timeout: {
+        alias: 't',
+        default: 2000,
+        description: 'Specify test timeout threshold (in milliseconds)',
+        group: GROUPS.RULES
+      },
+      ui: {
+        alias: 'u',
+        default: 'bdd',
+        description: 'Specify user interface',
+        group: GROUPS.RULES,
+        requiresArg: true,
+        type: 'string'
+      },
+      watch: {
+        alias: 'w',
+        description: 'Watch files in the current working directory for changes',
+        group: GROUPS.FILES,
+        type: 'boolean'
+      }
+    })
+    .positional('spec', {
+      default: ['test/'],
+      defaultDescription: 'test',
+      description: 'One or more files, directories, or globs to test',
+      type: 'array'
+    })
```

Comment:
Why doesn't positional show up last?

Reply:
In the help output?  The ungrouped options always show last...  I don't know if we have control over that.

Reply:
Shove them in a group?

Reply:
you mean put `spec` in a group, e.g., "File Handling"?